### PR TITLE
miage-rapide-tp:0.1.0

### PR DESCRIPTION
**Pull request by RemiSaurel**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Typst template to generate a practical work report for students of the MIAGE (Méthodes Informatiques Appliquées à la Gestion des Entreprises).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/miage-rapide-tp/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#import "@local/miage-rapide-tp:0.1.0": *
```
Comment (laurmaedje) [inline packages/preview/miage-rapide-tp/0.1.0/template/main.typ:1]:
```suggestion
#import "@preview/miage-rapide-tp:0.1.0": *
```

File: packages/preview/miage-rapide-tp/0.1.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+= TP
+
+Une mini introduction pour le TP...
+#question("La question 1 ?")
+#remarque("Remarque personnalisée", bg-color: olive, text-color: white)
+
+#question("Donne un exemple de code en python.")
+Voici le code nécessaire pour afficher "Hello, World!" en Python :
+#code_block("print('Hello, World!')", "py", title: "HelloWorld en Python")
```
Comment (laurmaedje) [inline packages/preview/miage-rapide-tp/0.1.0/template/main.typ:35]: The idiomatic casing convention in Typst is `kebab-case`, so I would recommend switching to that. But it is not a requirement.

File: packages/preview/miage-rapide-tp/0.1.0/template/main.typ (hunk 1 above)
Comment (RemiSaurel) [reply packages/preview/miage-rapide-tp/0.1.0/template/main.typ]: fixed in 5e20354fa3f87bbbfc94b9644421295b39e9b980

File: packages/preview/miage-rapide-tp/0.1.0/template/main.typ (hunk 2 above)
Comment (RemiSaurel) [reply packages/preview/miage-rapide-tp/0.1.0/template/main.typ]: sorry about that, fixed it in 5e20354fa3f87bbbfc94b9644421295b39e9b980 !

**Comment by RemiSaurel** [comment]
thanks for the review @laurmaedje, fixed everything in [5e20354](https://github.com/typst/packages/pull/943/commits/5e20354fa3f87bbbfc94b9644421295b39e9b980)

**Pull request merged**